lua/delaytrain/init.lua

```diff
@@ -46,6 +65,8 @@ function M.setup(opts)
             vim.g.delaytrain_grace_period = opts.grace_period
         end
 
+        ignore_filetypes = vim.tbl_extend("force", ignore_filetypes, opts.ignore_filetypes)
```

Comment:
Was the itention for `tbl_extend` to be able to add default ignored filetypes in the future and add to the list? Personally I'd rather leave those decisions up to the user.

This also may run into issues if a user removes an ignored filetype from their config and reloads it while nvim is running.

Reply:
You're right I didn't think about this scenario I make sure to only load user defined opts